packages/frontend/src/app/not-found/not-found.component.html

```diff
@@ -0,0 +1,27 @@
+<!--
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<div id="main-container">
+    <h1>404</h1>
+
+    <h3>Sorry, but the thing you are looking for isn't here. You can:</h3>
```

Comment:
Just a suggestion for wording in place of "the thing you are looking for isn't here": "The page you are looking for does not exist" or the "The page you are looking for is not found" ?